.gitpod.yml

```
@@ -1,24 +1,29 @@
 tasks:
-  - name: Deploy Jekyll Docs
-    command: export JEKYLL_ENV=production; cd /workspace/AaC/docs; bundle install && bundle exec jekyll s
+  - name: Serve Sphinx Docs
+    init: |
+      cd /workspace/AaC/docs/
+      sleep 120 # Gitpod often takes a long time to install the python environment
+      source /workspace/AaC/python/venv/bin/activate
+      sphinx-build source gen_docs # Pre-build the docs so that when Gitpod opens the port it's ready to serve
+    command: sphinx-autobuild source gen_docs
 
   - name: VSCode Extension Environment
     init: |
-      cd vscode_extension/
+      cd /workspace/AaC/vscode_extension/
       yarn install
+      sleep 120 # Gitpod often takes a long time to install the python environment
+      source /workspace/AaC/python/venv/bin/activate
+      aac gen-openapi-spec
+      yarn generate:rest_api
```

The `yarn` command will generate the OpenAPI spec (additionally, `gen-openapi-spec` requires an argument so this won't work, anyways). See my PR #659 where I update the `yarn` command to add the extra argument.

```suggestion
      yarn generate:rest_api
```